bolt-modules/out/lib/puppet/functions/out/verbose.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# Output a message for the user when running with --verbose
```

Comment:
```suggestion
# Output a message for the user when running in verbose mode.
```

Plans aren't always run through the Bolt CLI, so it doesn't make as much sense to use a CLI flag in the description.